src/cmd/longevity_test/longevity.go

```diff
@@ -73,6 +73,12 @@ func pitrListBackups(
 	pitr time.Time,
 	backupIDs []string,
 ) error {
+	logAndPrint(
+		ctx,
+		"looking for %d backup(s) using timestamp %v\n",
```

Comment:
using/at/since?  backup/latest backup?
```suggestion
		"looking for %d backup(s) using timestamp %v\n",
```

Reply:
timestamp is relative to when deletion happened not the time of the backup(s)